__tests__/reporters/__snapshots__/json.test.ts.snap

```diff
@@ -10,64 +10,34 @@ Object {
   "ecs": Object {
     "http": Object {
       "request": Object {
-        "body": Object {
-          "bytes": 0,
-          "content": "",
-        },
         "headers": Object {
           "accept_encoding": "gzip, deflate, br",
           "method": "GET",
           "path": "/",
           "upgrade_insecure_requests": "1",
           "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) HeadlessChrome/90.0.4392.0 Safari/537.36",
         },
-        "initial_priority": "VeryHigh",
         "method": "GET",
-        "mixed_content_type": "none",
-        "referrer": undefined,
-        "referrer_policy": "strict-origin-when-cross-origin",
         "url": "https://vigneshh.in/",
       },
       "response": Object {
-        "body": Object {
-          "bytes": 3329,
-        },
-        "connection_id": 12,
-        "connection_reused": false,
```

Comment:
I take it all these fields are unused currently?

Reply:
Yes thats correct, we were directly sending all the fields that were coming out of chrome protocol. https://chromedevtools.github.io/devtools-protocol/tot/Network/#type-Response

Now we only send the strict data that is being given out from PW.